Dockerfile.builder

```diff
@@ -16,31 +13,51 @@ RUN set -ex \
     B9AE9905FFD7803F25714661B63B535A4C206CA9 \
     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
   ; do \
-    gpg --import "/gpg/${key}.gpg" ; \
```

Comment:
@naina-verma any reason why this changed in your branch ? I just took what you had in your branch.